enterprise/backend/src/metabase_enterprise/enhancements/integrations/ldap.clj

```diff
@@ -30,11 +30,6 @@
   :default "userPassword,dn,distinguishedName"
   :type    :csv)
 
-(defsetting ldap-sync-admin-group
```

Comment:
by removing this, it'll also remove this option on Frontend because the front end displays a list of settings returned by the backend.

Reply:
We have a list of `retired-setting-names` in `metabase.models.setting` that ensures that setting names aren't reused after they are removed (in case values are still lingering in DBs). Could you add this there?